zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookSocket.java

```diff
@@ -67,7 +67,7 @@ public String getProtocol() {
   }
 
   public synchronized void send(String serializeMessage) throws IOException {
-    connection.getRemote().sendStringByFuture(serializeMessage);
+    connection.getRemote().sendString(serializeMessage);
```

Comment:
This line should not be part of this PR.

Reply:
Fixed